src/rosduct/rosduct_impl.py

```diff
@@ -204,6 +175,117 @@ def initialize(self):
                 param = param[0]
             self.last_params[param] = self.client.get_param(param)
 
+    def add_local_topic(self, msg):
+        bridgepub = self.client.publisher(msg.alias_name,
+                                          msg.conn_type,
+                                          latch=msg.latch)
```

Comment:
@marc-hanheide 
Getting an error here (see below) because no value has been set for msg.latch.  May be this value should be read from the config file and set accordingly at [L117-L122] (https://github.com/LCAS/rosduct/blob/1a8f8fb63884fda00acfd19b0189709f306a4893/src/rosduct/rosduct_impl.py#L117-L122) and [L130-L136](https://github.com/LCAS/rosduct/blob/1a8f8fb63884fda00acfd19b0189709f306a4893/src/rosduct/rosduct_impl.py#L130-L136).  

`Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/rosduct/rosduct_main.py", line 8, in <module>
    r = ROSduct()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rosduct/rosduct_impl.py", line 87, in __init__
    self.initialize()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rosduct/rosduct_impl.py", line 141, in initialize
    self.add_local_topic(msg)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rosduct/rosduct_impl.py", line 181, in add_local_topic
    latch=msg.latch)
AttributeError: 'ROSDuctConnection' object has no attribute 'latch'`

Reply:
You are right... I don't know how this slipped in. Maybe fix locally and open PR?